confidant/models/credential.py

```diff
@@ -52,6 +52,11 @@ class Meta:
     modified_by = UnicodeAttribute()
     documentation = UnicodeAttribute(null=True)
 
+    # Classification info (eg: FINANCIALLY_SENSITIVE)
+    category = UnicodeAttribute(null=True)
```

Comment:
Can we do:

```
tags = ListAttribute(default=list)
```

Reply:
Oh wait, I thought we were going to add tags in settings.py or as env vars?